ui/app/reports/views/reports.html

```diff
@@ -5,12 +5,21 @@ <h2 class="section-title">{{::'REPORTS_TITLE_KEY' | translate}}</h2>
         <table class="alt-row form">
             <thead>
             <tr>
-                <th>{{::'REPORTS_NAME_HEADER_KEY' | translate}}</th>
+                <th style="display: flex; justify-content: space-between;">
```

Comment:
The `<th> ` tag seems to be missing its closing tag, which could be causing uglify error. Please check this.